index.ts

```diff
@@ -5,6 +5,7 @@ const baseUrl = `api.airtable.com/v0`;
 type Method = "GET" | "POST" | "PUT" | "DELETE" | "PATCH";
 
 state.webhooks = state.webhooks ?? {};
+state.webhooksTimers = state.webhooksTimers ?? [];
```

Comment:
```suggestion
state.webhooksTimers = state.webhooksTimers ?? {};
```
Shouldn't this be an object?